omc3/harpy/constants.py

```diff
@@ -10,6 +10,8 @@
 FILE_FREQS_EXT = ".freqs{plane}"
 FILE_LIN_EXT = ".lin{plane}"
 
+LINFILES_SUBFOLDER = "lin_files"
+
```

Comment:
Since we're modifying this file, shall we type hint everyone there as `str`? It helps the IDEs give correct information when we're deep into the code somewhere else.